src/compiler/abstract_compiler.nit

```diff
@@ -1506,6 +1506,46 @@ abstract class AbstractCompilerVisitor
 		return res
 	end
 
+	# Generate a int8 value
+	fun int8_instance(value: Int8): RuntimeVariable
+	do
+		var t = mmodule.int8_type
+		var res = new RuntimeVariable("((int8_t){value.to_s})", t, t)
+		return res
+	end
+
+	# Generate a int16 value
+	fun int16_instance(value: Int16): RuntimeVariable
+	do
+		var t = mmodule.int16_type
+		var res = new RuntimeVariable("((int16_t){value.to_s})", t, t)
+		return res
+	end
+
+	# Generate a uint16 value
```

Comment:
I'm skeptical about this one, if I trust my gut, it tells me not to use `an` in this case